cpp/src/structs/structs_column_view.cpp

```diff
@@ -25,6 +25,8 @@ structs_column_view::structs_column_view(column_view const& rhs) : column_view{r
   CUDF_EXPECTS(type().id() == type_id::STRUCT, "structs_column_view only supports struct columns");
 }
 
+column_view structs_column_view::parent() const { return *this; }
```

Comment:
Completely unrelated to this PR but I am not a fan of this idiom that `parent()` returns essentially "me".  It doesn't line up with how you'd normally think of the idea of a parent in any kind of data structure. `base()` might be better, or something along those lines.

Reply:
> Completely unrelated to this PR but I am not a fan of this idiom that `parent()` returns essentially "me". It doesn't line up with how you'd normally think of the idea of a parent in any kind of data structure. `base()` might be better, or something along those lines.

This is discussed in #9111
I don't want to start another discussion on this in this PR.

Reply:
> `base()` might be better...

For the record and what little it's worth, I agree with you, @nvdbaranec. :]

Let's ~fight~ engage in gentlemanly deliberation in #9111.